src/mame/jaleco/megasys1.h

```diff
@@ -14,6 +14,7 @@
 
 #include "machine/gen_latch.h"
 #include "machine/timer.h"
+#include "cpu/tlcs90/tlcs90.h"
 #include "sound/msm5205.h"
 #include "sound/okim6295.h"
 #include "ms1_tmap.h"
```

Comment:
Please keep headers in alphabetical order for a module, and put local project headers before device headers.